lib/src/_isolates_io.dart

```diff
@@ -11,8 +11,10 @@ class PostgrestIsolate {
   late final SendPort _sendPort;
   final _createdIsolate = Completer<void>();
   late final _events = StreamQueue(_receivePort);
+  bool _startedInit = false;
```

Comment:
Maybe we can rename this variable to something like this?
```suggestion
  bool _hasBeenInitialized = false;
```

Reply:
The initialization is async and wanted to make clear that the init is not garanted to be finished. Your change could mean that it finished with init, but that's what the completer is for.

Reply:
I see! Then maybe `_hasStartedInit`?

Reply:
:+1: fixed